### PR TITLE
Only rename 'UIA' if it's in an XPath's axis

### PR DESCRIPTION
**Pull request by dpgraham**
(see issue https://github.com/appium/appium-xcuitest-driver/issues/270)

To test if 'UIA' is in the axis of an xpath check if it has a forward slash preceding it or if it is at the beginning of the selector string

### CODE REVIEW

**Comment by dpgraham** [comment]
@jlipps @imurchie

File: lib/commands/find.js [hunk 1]
```diff
@@ -54,7 +54,14 @@ helpers.findNativeElementOrElements = async function (strategy, selector, mult,
...
   if (strategy === 'xpath') {
-    selector = selector.replace(/(UIA)([^\[\/]+)/g, (str, g1, g2) => {
+    // Replace UIA if it comes after a forward slash
+    selector = selector.replace(/\/(UIA)([^\[\/]+)/g, (str, g1, g2) => {
+      rewroteSelector = true;
+      return '/' + stripViewFromSelector(`XCUIElementType${g2}`);
+    });
+
+    // Replace UIA if it is at the beginning of the selector string
+    selector = selector.replace(/^(UIA)([^\[\/]+)/, (str, g1, g2) => {
```
Comment (rjalmo) [inline lib/commands/find.js]: Can these two regexes be reduced to a single expression? A regex that produces similar results (replacing "UIA" both after a forward slash and beginning of a string) can be:
```
selector = selector.replace(/(^|\/)(UIA)([^\[\/]+)/, (str, g1, g2, g3) => {
  rewroteSelector = true;
  return '/' + stripViewFromSelector(`XCUIElementType${g2}`);
});
```

**Comment by dpgraham** [comment]
Thanks @rjalmo. I've amended the branch to include your suggestion. It looks like this now:

selector = selector.replace(/(^|\/)(UIA)([^\[\/]+)/g, (str, g1, g2, g3) => {
      rewroteSelector = true;
      return g1 + stripViewFromSelector(`XCUIElementType${g3}`);
    });

**Comment by jlipps** [comment]
published as 2.0.32

**Pull request merged**